c/include/libsbp/bootload.h

```diff
@@ -50,6 +50,29 @@ SBP_PACK_START
  * protocol version number.
  */
 #define SBP_MSG_BOOTLOADER_HANDSHAKE_RESP  0x00B4
+#define SBP_BOOTLOADER_HANDSHAKE_RESP_SBP_MAJOR_PROTOCOL_VERSION_NUMBER_MASK (0xff)
```

Comment:
We should think about alternate ways to generate these names since these generated names are really long.  We could consider on of the following:
- prefix the name with `FIELD_` or `BOOTLOAD_FIELD_`, and only generate the longer name if there's a conflict -- this makes the names not as discover-able via completion but much shorter to type, so not sure I really like this idea
- generate a enum with the bitfield values, the name would still be long, but it'd at least be split over the enum type e.g. `sbp_bootloader_handshake_resp.sbp_major_protocol_version_number_mask` -- the downside of this would be the need to cast to an appropriate type during use
- add field to the spec to specify a symbolic name for the field instead of generating it from the description

Additionally, we could improve the typing of this with the following change:
- generate a typedef'd enum that identifies each bitfield
- generate an (inlined) function which takes the bitfield, a bitfield enum, then returns the requested value (instead of using GET/SET macros here)